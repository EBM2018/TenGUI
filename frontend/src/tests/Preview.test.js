import ActTextCheck from '../react/FishtankAdmin/Preview/Preview';
import React from 'react';
import renderer from 'react-test-renderer';

test ('derp',()=>{
    expect('True').toBeTruthy;
});
