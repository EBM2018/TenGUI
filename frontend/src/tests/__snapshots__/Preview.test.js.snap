// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does this render 1`] = `
<d>
  <div
    id="title"
  >
    Prévisulasation de l'activité
  </div>
  <div
    id="subtitle"
  >
    Description
  </div>
  <div
    id="description"
  >
    blabla
  </div>
  <div
    id="subtitle"
  >
    Effet
  </div>
  <div
    id="effect"
  >
    blabla
  </div>
  <div
    id="subtitle"
  >
    Affichage
  </div>
  <div
    id="display"
  >
    blabla
  </div>
</d>
`;
